base/applications/rapps/appview.cpp

```diff
@@ -1782,7 +1782,7 @@ VOID CApplicationView::OnCommand(WPARAM wParam, LPARAM lParam)
 {
     if (lParam)
     {
-        if ((HWND)lParam == m_SearchBar->GetWindow())
+        if ((HWND)lParam == m_SearchBar->m_hWnd)
```

Comment:
```suggestion
        if (reinterpret_cast<HWND>(lParam) == m_SearchBar)
```

Reply:
The reinterpret cast on the scalar lParam is not necessary.